File: InJava/app/src/main/java/com/barryzeha/quizquestion/view/MainActivity.java
package com.barryzeha.quizquestion.view;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;

import com.barryzeha.quizquestion.R;
import com.barryzeha.quizquestion.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {
    private ActivityMainBinding bind;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        bind=ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(bind.getRoot());
        setUpListeners();
    }
    private void setUpListeners(){
        bind.btnPlay.setOnClickListener(view-> startActivity(new Intent(MainActivity.this, QuizActivity.class)));
        bind.btnSeeRanking.setOnClickListener(v-> startActivity(new Intent(MainActivity.this,RankingActivity.class)));
    }
}

File: InKotlin/app/src/main/java/com/barryzeha/quizquestion/view/MainActivity.kt
package com.barryzeha.quizquestion.view

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.barryzeha.quizquestion.R
import com.barryzeha.quizquestion.common.loadUrl
import com.barryzeha.quizquestion.common.startActivity
import com.barryzeha.quizquestion.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var bind: ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        bind= ActivityMainBinding.inflate(layoutInflater)
        setContentView(bind.root)

        setUpListeners()

    }


    private fun setUpListeners() {
        bind.ivBackgroundMain.loadUrl(R.drawable.main_wallpaper)
        bind.btnPlay.setOnClickListener { startActivity<QuizActivity> {} }
        bind.btnSeeRanking.setOnClickListener{startActivity<RankingActivity> {}}
    }
}